src/scripts/main.js

```diff
@@ -1,3 +1,83 @@
 'use strict';
 
-// write your code here
+const BASE_URL
+= 'https://mate-academy.github.io/phone-catalogue-static/api/phones';
+const url = `${BASE_URL}.json`;
+
+const body = document.querySelector('body');
+const listOfPhones = document.createElement('ul');
+const description = document.createElement('div');
+const title = document.createElement('h2');
+const titleDescr = document.createElement('h2');
+
+listOfPhones.className = 'list';
+description.className = 'list';
+
+title.innerHTML = 'Phones List:';
+titleDescr.innerHTML = 'Description:';
+
+body.insertAdjacentElement('afterbegin', listOfPhones);
+body.insertAdjacentElement('beforeend', description);
+listOfPhones.insertAdjacentElement('afterbegin', title);
+description.insertAdjacentElement('afterbegin', titleDescr);
+
+const getPhones = () => {
+  return fetch(url)
+    .then(response => {
+      if (!response.ok) {
+        setTimeout(() => {
+          throw new Error(`${response.status} - ${response.statusText}`);
+        }, 5000);
+      }
+
+      return response.json();
+    });
+};
+
+const getPhonesDetails = (phone) => {
```

Comment:
```suggestion
const getPhonesDetails = (phones) => {
```
Это же все еще массив